website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
+
+<!--Confirm how users can get this feature-->
+:::info Beta feature 
+If you are interested in getting access to this feature, please contact us. 
+:::
+
+User-defined functions (UDFs) enable users to define and register custom functions within the warehouse. Like [macros](/docs/build/jinja-macros), UDFs enable reuse of code; however, unlike macros, you can define UDFs in languages other than SQL (for example, Python, Java, Scala) and you can use them in queries outside the <Constant name="dbt_platform" />.
+
+Note that only basic SQL UDFs are currently supported in dbt.<!--Is this correct?-->
+
+dbt creates, updates, and renames UDFs as part of DAG execution. The UDF file is created before building the model that references it.
+
+## Supported adapters
+
+UDFs are supported in the following adapters:
+
+- BigQuery
+- Snowflake
+- Redshift
```

Comment:
And postgres! Though whether that is important to document 🤷🏻 don't know how many people use it in cloud